pkg/allocator/allocator_test.go

```diff
@@ -340,7 +340,7 @@ func testAllocator(c *C, maxID idpool.ID, allocatorName string, suffix string) {
 	rateLimiter := rate.NewLimiter(10*time.Second, 100)
 
 	// setup operator metrics, RunGC() relies on this
-	opoption.Config.OperatorPrometheusServeAddr = ":6942"
+	opoption.Config.OperatorPrometheusServeAddr = opoption.PrometheusServeAddr
```

Comment:
Can you squash the changes from this commit into the previous commit? There's no need to have a commit that hardcodes "6942" and then a new commit that makes it a variable. It makes more sense to just simplify the diff to the change that makes `OperatorPrometheusServeAddr` a variable.

Reply:
Sorry for the inconvenience, squashed them into one commit.